build_tools/ci/cpu_comparison/run.py

```diff
@@ -1122,6 +1182,68 @@ def __init__(self):
                 )
             )
 
+        self.register(
+            Matmul(
+                256,
+                256,
+                256,
+                "bf16",
+                "f32",
+                name_suffix="air_pad_pack",
+                use_ukernel=True,
+                lower_to_aie_pipeline="air",
+                tile_pipeline="pad-pack",
+            )
+        )
+
+        # M, K, N (copies from run_matmul_tests.sh)
+        bf16_ukernel_shapes_medium = [
+            [256, 256, 256],
+            [128, 512, 512],
+            [512, 4096, 2048],
+        ]
+        for shape in bf16_ukernel_shapes_medium:
+            self.register(
+                Matmul(
+                    shape[0],
+                    shape[2],
+                    shape[1],
+                    "bf16",
+                    "f32",
+                    use_ukernel=True,
+                    lower_to_aie_pipeline="objectFifo",
+                    tile_pipeline="pack-peel",
+                    n_repeats=2,
+                )
+            )
+
+        self.register(
+            Matmul(
+                32,
+                32,
+                32,
+                "i32",
+                "i32",
+                name_suffix="chess",
+                use_chess=True,
+                n_repeats=10,
+            )
+        )
+
+        self.register(
```

Comment:
Add comments for these tests as well. It will be easier for users to quickly get the purpose of the test.

Reply:
Can you suggest a comment, please?

Reply:
// ukernel test for chess